themes/classic/templates/catalog/_partials/miniatures/product.tpl

```diff
@@ -78,9 +78,9 @@
                 {/if}
               </span>
 
-              <div itemprop="offers" itemscope itemtype="https://schema.org/Offer" class="invisible">
-                <meta itemprop="priceCurrency" content="{$currency.iso_code}" />
-                <meta itemprop="price" content="{$product.price_amount}" />
+              <div class="invisible">
+                <meta content="{$currency.iso_code}" />
+                <meta content="{$product.price_amount}" />
               </div>
```

Comment:
```suggestion
              <div class="invisible">
                <meta content="{$currency.iso_code}" />
                <meta content="{$product.price_amount}" />
              </div>
```

The same as above ?

Reply:
Yes, Same as above do I remove the complete div block 
```
<div class="invisible">
  <meta content="{$currency.iso_code}" />
  <meta content="{$product.price_amount}" />
</div>
```

Reply:
Meta removed too in 844910e